lib/iris/tests/unit/analysis/test_PERCENTILE.py

```diff
@@ -196,6 +196,13 @@ def test_missing_mandatory_kwarg(self):
         with self.assertRaisesRegex(ValueError, emsg):
             PERCENTILE.aggregate("dummy", axis=0)
 
+    def test_wrong_kwarg(self):
```

Comment:
This seems like a common user error - do we want a more informative message or are we happy that the docstring will help people enough?

Reply:
The full message is "mquantiles() got an unexpected keyword argument 'method'" but for some reason the assertion failed with a bad match when I pasted the lot in 😕

Reply:
Probably the brackets needing some form of escaping, seems fine as-is though